### ISSUE TITLE
Bring the project back

### ISSUE THREAD

**Issue by jakiisheep** [TonCherAmi/watch#2]
I just love this project, and I would love it if you could still maintain this, even tho the last commit was in 2017, the idea for the project and the project itself is amazing, I would love to see more features.

**Comment by TonCherAmi** [comment]
Eh? Are you sure you have the right repository? This is rather unexpected :)
To be honest I've stopped using tvsm a while ago so I'm not sure how much time I would be able to devote to maintaining it, but nevertheless, I'm curious, what kind of features did you have in mind?

**Comment by jakiisheep** [comment]
Yes, I have the right repo :) Well, I don't really know myself which features it needs, but it sure needs an update, but it seems like a really good project that does have potential!

**Comment by jakiisheep** [comment]
I was just casually browsing GitHub and clicked on Polybar's website and clicked on your GitHub profile which was linked in the footer as "Design"

**Comment by TonCherAmi** [comment]
What kind of an update? If neither of us knows then there's hardly something to work on here :)
I suppose I could fix it if it was broken but I've tried it out just now and seems to work still. You saying that it has potential is nice and all but it's had approximately zero users for the last couple of years and I've little interest in trying to promote it really. If you encounter any issues do let me know I'll try taking a look if I can find the time.

**Comment by jakiisheep** [comment]
> What kind of an update? If neither of us knows then there's hardly something to work on here :) I suppose I could fix it if it was broken but I've tried it out just now and seems to work still. You saying that it has potential is nice and all but it's had approximately zero users for the last couple of years and I've little interest in trying to promote it really. If you encounter any issues do let me know I'll try taking a look if I can find the time.

I can promote it (also sorry for the late response) but it just came to my mind
